code/segmentedtp/stplog.q

```diff
@@ -0,0 +1,96 @@
+// Utilites for periodic tp logging in stp process
+
+.proc.loadf[raze .proc.params`stpconfig]
+
+\d .stplg
+
+d:.z.d
+t:tables`.
+
+// Create stp log directory
+// Log structure `:stplogs/date/tabname_time
+createdld:{[name;date]
+  .os.md dldir::hsym`$getenv[`TORQHOME],(,/)"/stplogs/",string name,date
```

Comment:
I would possible require a separate environment variable for this, and exit if it's not set. It probably definitely shouldn't live in TORQHOME, there should be a DATA directory somewhere for this stuff